eclair-core/src/main/resources/reference.conf

```diff
@@ -76,6 +77,7 @@ eclair {
   // NB: this number effectively reduces the expiry-delta-blocks, so you may want to take that into account and increase
   // expiry-delta-blocks.
   fulfill-safety-before-timeout-blocks = 24
+  min-final-expiry-delta-blocks = 30 // Bolt 11 invoice's min_final_cltv_expiry; must be strictly greater than fulfill-safety-before-timeout-blocks
```

Comment:
The difference between `expiry-delta-blocks`, `fulfill-safety-before-timeout-blocks`, `min-final-expiry-delta-blocks` is pretty subtle, maybe we should prefix them with `relay` or `receive` when applicable? Can be revamped in a later PR though.

Reply:
Agreed, I think it's worth a separate PR to put them in a dedicated subsection of the config (which requires some tests in `StartupSpec` to guide users on how to change their config).